src/player.cpp

```diff
@@ -3773,22 +3773,26 @@ void player::disp_status(WINDOW *w, WINDOW *w2)
   int speedoy = sideStyle ? 5 :  3;
 
   bool metric = OPTIONS["USE_METRIC_SPEEDS"] == "km/h";
-  const char *units = metric ? _("km/h") : _("mph");
+  // Logic below is not applicable to translated units and should be changed
   int velx    = metric ? 4 : 3; // strlen(units) + 1
   int cruisex = metric ? 9 : 8; // strlen(units) + 6
-  float conv  = metric ? 0.0161f : 0.01f;
 
-  if (0 == sideStyle) {
-    if (!veh->cruise_on) speedox += 2;
-    if (!metric)         speedox++;
-  }
+    if( !sideStyle ){
```

Comment:
I hope that I didn't brake previous logic here, I don't like these c++ casting tricks.